auth-web/src/components/pay/eft/ShortNamePaymentHistory.vue

```diff
@@ -152,6 +153,7 @@ import { DEFAULT_DATA_OPTIONS } from '../../datatable/resources'
 import { EFTTransactionState } from '@/models/eft-transaction'
 import ModalDialog from '@/components/auth/common/ModalDialog.vue'
 import PaymentService from '@/services/payment.services'
+import { Vue } from 'vue-property-decorator'
```

Comment:
Not too sure why we're introducing this, we're actively trying to get away from using Vue-class-component and vue-property-decorartor

Reply:
`import Vue from 'vue'`

Reply:
Will fix this in my next PR, as it is another FE ticket.

Reply:
all good, I was too slow on the review haha